components/Tabs/src/TabBarView/MDCTabBarView.h

```diff
@@ -56,6 +75,13 @@ __attribute__((objc_subclassing_restricted)) @interface MDCTabBarView : UIScroll
  */
 @property(nullable, nonatomic, copy) UIColor *selectionIndicatorStrokeColor;
 
+/**
+ The preferred layout style of the bar.  If possible, this layout style will be used. If not,
+ another style will be used. For example, if the bar is too narrow for a Fixed layout style, then a
+ Scrollable layout style may be used instead.
```

Comment:
Please document which layout style the Tab Bar View defaults to if this isn't set, _Rob_.

Reply:
Done.